helm/values.yaml

```diff
@@ -32,6 +32,9 @@ secret:
     # This secret is a random string used to authenticate to the CAS service.
     # This can be any string you care to use generated by any mechanism you
     #   prefer.
+    # You could use something like:
+    #  `cat /dev/urandom | LC_CTYPE=C tr -cd '[:graph:]' | head -c 32`
+    #  to generate this string.
```

Comment:
it's probably worth noting that we suggest a different random string creation mechanism about 12 lines up.

This is just the one I found when I googled "bash generate random string"